pypeit/wavecalib.py

```diff
@@ -741,6 +764,97 @@ def build_wv_calib(self, arccen, method, skip_QA=False,
         self.steps.append(inspect.stack()[0][3])
         return self.wv_calib
 
+    def redo_echelle_orders(self, bad_orders:np.ndarray, dets:np.ndarray, order_dets:np.ndarray):
+        """ Attempt to redo the wavelength calibration for a set 
+        of bad echelle orders
+
+        Args:
+            bad_orders (np.ndarray): Array of bad order numbers
```

Comment:
Is this the correct format for a docstring? Do you need the type if you're using PEP484 annotations? Also, the order_dets starts on a new line (unlike the other ones)... I'm not the best person to ask, but I flag it just in case...

Reply:
yes, this is Google format